### DISCUSSION TITLE
Reading Comments with break line Ods Format

### DISCUSSION THREAD

**Discussion by MarioGattolla** [PHPOffice/PhpSpreadsheet#4081] [Q&A]
Hi i'm tryng to import and read a comment from a cell like the example

![Schermata del 2024-07-01 10-40-42](https://github.com/PHPOffice/PhpSpreadsheet/assets/94918437/df9c7740-e679-4d85-b5a9-fc4ec3aa0c89)

When i try to read the comment the method return me this:

![Schermata del 2024-07-01 10-42-12](https://github.com/PHPOffice/PhpSpreadsheet/assets/94918437/913c23e4-9011-4767-9eec-8a8bf893ba24)

it seems that the method is reading all before the first break line and is ignoring all after. 

Any tips?  Im actually using a .odt file and the cell has no formatting/style

**Comment by oleibman** [comment]
Confirmed - this is a bug in the Ods Reader. I will try to submit a PR to fix it in the next day or two.

**Reply by MarioGattolla** [reply]
Thank you very much, do you need i open a dedicate issue to connect the PR?

**Reply by oleibman** [reply]
I think this discussion will do; no need to open an issue as well. PR #4086 should address your problem. If you have the opportunity to test with that, please do so.

**Reply by MarioGattolla** [reply]
Tested with you PR and it's working now

**Reply by oleibman** [reply]
Thank you for confirming.

**Comment by oleibman** [comment]
PR #4086 has now been merged and will fix this problem.
